06-building-a-program-start-to-finish/solving-the-problem/Output/invertedindex.js

```diff
@@ -0,0 +1,156 @@
+/**
```

Comment:
Your JSDoc used in this file doesn't properly describe your method/functions. Please address this.